autoprotocol/container.py

```diff
@@ -375,6 +376,47 @@ def set_volume(self, vol):
         self.volume = v
         return self
 
+    def set_compounds(self, compounds):
+        """
+        Sets the list of associated compounds, and their metadata, to an aliquot.
+
+        Parameters
+        ----------
+        compounds : list
+            List of compounds associated to a well.
+
+        Returns
+        -------
+        Well
+            Well with associated compounds
+
+        Raises
+        ------
+        TypeError
+            Incorrect input-type given
+        RuntimeError
+            Compound information does not contain required keys
+        """
+        expected_keys = {"id", "molecularWeight", "smiles"}
```

Comment:
Update the key as well.

Reply:
So idk if I should do that here... we're getting these keys from web, then this function transforms `molecularWeight` --> `molecular_weight`

Reply:
Gotcha. All good then.